src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_ObjectOrCollectionInitializerExpression.cs

```diff
@@ -16,13 +16,14 @@ internal sealed partial class LocalRewriter
     {
         private static BoundObjectInitializerExpressionBase UpdateInitializers(BoundObjectInitializerExpressionBase initializerExpression, ImmutableArray<BoundExpression> newInitializers)
         {
-            if (initializerExpression.Kind == BoundKind.ObjectInitializerExpression)
+            switch (initializerExpression)
```

Comment:
Nit: could use a switch expression

Reply:
Yes, I considered that but decided on a switch statement to match the method below.

---
In reply to: [325892063](https://github.com/dotnet/roslyn/pull/38732#discussion_r325892063) [](ancestors = 325892063)